src/details/ArborX_Predicates.hpp

```diff
@@ -12,6 +12,7 @@
 #define ARBORX_PREDICATE_HPP
 
 #include <ArborX_DetailsAlgorithms.hpp>
+#include <ArborX_Polytope.hpp>
```

Comment:
Why is this needed?

Reply:
Need to see geometric algorithms

Reply:
The alternative is `using Details::intersects` here 
https://github.com/arborx/ArborX/blob/26467f9a09dbb8db4bfd33bac0aa38172f36bdc0/src/details/ArborX_Predicates.hpp#L62